src/features/tags/components/TagItem.js

```diff
@@ -1,20 +1,37 @@
 import React, { Component } from "react";
 import { connect } from "react-redux";
+import { NavLink } from 'react-router-dom';
 import { getTagById } from "../selectors";
 import { Link } from "@mui/material";
 
 class TagItem extends Component {
   render() {
     const { tag, key } = this.props;
-
+    
     return (
-      <li className="menu__list-item" name={tag.id} key={key}>
+      <li 
+        className="menu__list-item" 
```

Comment:
This is still dependent on a CSS class. Check out the [List Item](https://mui.com/api/list-item/) and [Typography](https://mui.com/components/typography/) components to give the list a similar look as before:

- grey text
- the whole TagItem background would turn light grey on hover
- no underline on text, even on hover